config/horizon.json

```diff
@@ -116,7 +116,8 @@
 		"wpcom-user-bootstrap": true,
 		"wordpress-action-search": false,
 		"redirect-fallback-browsers": true,
-		"woop": false
+		"woop": false,
+		"onboarding/import-light-url-screen": true
```

Comment:
I think we could move this line a bit higher - to keep the flags in the alphabetical order (similarly to how we have it with `signup/goals-step-2`).

Reply:
Let's do it 👍